src/theory/quantifiers/ce_guided_pbe.cpp

```diff
@@ -1502,7 +1503,7 @@ Node CegConjecturePbe::constructBestSolvedConditional( std::vector< Node >& solv
 Node CegConjecturePbe::constructBestConditional( std::vector< Node >& conds, UnifContext& x ) {
   Assert( !conds.empty() );
   // TODO
-  double r = (double)(rand())/((double)(RAND_MAX));
+  double r = (double)(RNG::getRNG().rand())/((double)(UINT64_MAX));
```

Comment:
This seems like a case for `pickDouble`.

Reply:
You're right, seems that I missed this because I added pickDouble after making these changes...